graphql/resolve/mutation.go

```diff
@@ -98,7 +98,7 @@ type MutationRewriter interface {
 	//     }
 	// These query will be created in case of Add or Update Mutation which references node
 	// 0x127 or Computer of name "computer1"
-	RewriteQueries(ctx context.Context, m schema.Mutation) ([]*gql.GraphQuery, error)
+	RewriteQueries(ctx context.Context, m schema.Mutation) ([]*gql.GraphQuery, []schema.Type, error)
```

Comment:
I don't understand the need for returning []schema.Type from RewriteQueries. As far as I can see only the name of the type is used in mutation.go. Let's only return an array of type names from here.
Also update the comment above which describes what RewriteQueries does.

Reply:
Yeah, correct. Thanks.